src/core/mod.rs

```diff
@@ -646,7 +652,7 @@ enum_from_primitive! {
         GlContextKhr = ffi::CL_GL_CONTEXT_KHR as isize,
         EglDisplayKhr = ffi::CL_EGL_DISPLAY_KHR as isize,
         GlxDisplayKhr = ffi::CL_GLX_DISPLAY_KHR as isize,
-        CglSharegroupKhr = ffi::CL_CGL_SHAREGROUP_KHR as isize,
+        CglSharegroupKhr = CL_CGL_SHAREGROUP_KHR_OS_SPECIFIC as isize, // FIXME that's ugly
```

Comment:
Leave it ugly for now. We can rename it later if necessary without too much trouble.